src/components/FullDetailView/FullDetailPage.js

```diff
@@ -130,38 +187,39 @@ export class FullDetailPage extends Component {
         centered
         destroyOnClose={true}
         visible={visible}
-        onOk={this.handleSave}
-        onCancel={this.handleClose}
         width={'85%'}
```

Comment:
can we also add some top and bottom margin? Without it feels a bit odd having it fill the vertical space as well.